client/gui/SDL_Extensions.h

```diff
@@ -152,6 +152,57 @@ struct ColorPutter
 
 typedef void (*BlitterWithRotationVal)(SDL_Surface *src,SDL_Rect srcRect, SDL_Surface * dst, SDL_Rect dstRect, ui8 rotation);
 
+class ColorShifter
+{
+public:
+	virtual SDL_Color shiftColor(SDL_Color clr) = 0;
+};
+
+class ColorShifterLightBlue : public ColorShifter
+{
+public:
+	static ColorShifterLightBlue* create()
+	{
+		return new ColorShifterLightBlue();
+	}
+
+	SDL_Color shiftColor(SDL_Color clr) override
+	{
+		clr.b = clr.b + (255 - clr.b) / 2;

```

Comment:
This method should be `const` (and `ColorShifter` params where it used should be `const` too)

Reply:
I think this comment is in wrong place, since it modified the color.